### PR TITLE
Ex 30 user files request for permit

### PR DESCRIPTION
**Pull request by JMStewart00**
**1. What is this for?** Wanted to change the look and feel of the Permit process, plus change the way it worked based on feedback from client. 
**2. What caused the issue?** Feedback from the client.
**3. What was your approach?** We somehow lost the number we were using for comparison so added that back into the backend. Changed up the look of the modal based on the current_exactions number to show a danger message (when they shouldn't be adding a Permit) and a success message when they could.

### CODE REVIEW

File: lfucg-exactions/client/js/components/LotForm.js [hunk 1]
```diff
@@ -233,22 +233,28 @@ class LotForm extends React.Component {
...
+                                                {!submitEnabled ? (
+                                                    <div>
+                                                        <div className="clearfix" />
+                                                        <span> * All required fields must be filled.</span>
+                                                    </div>
+                                                ) : null
+                                                }
+                                            </div>
+                                            <div className="col-sm-6">
+                                                {currentLot && currentLot.lot_exactions && currentLot.lot_exactions.current_exactions_number > 0 &&
```
Comment (KellyInnovation) [inline lfucg-exactions/client/js/components/LotForm.js]: This red block could be overlooked.  Can we use onBlur and either open a modal or populate the same message in a line by itself in larger font?

**Comment by JMStewart00** [comment]
We can do whatever ya want! I can see what that looks like.

File: lfucg-exactions/client/js/components/LotSummary.js [hunk 2]
```diff
@@ -154,26 +154,45 @@ class LotSummary extends React.Component {
...
+                                                    <h3 className="modal-title text-center" id="modalLabel">Permit Addition</h3>
                                                 </div>
                                                 <div className="modal-body">
-                                                    <h4 className="text-center">Our records indicate an outstanding balance of <strong>{(currentLot.lot_exactions.current_exactions).toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')}</strong> for this lot. Please contact finance to submit payment prior to applying for a permit for:</h4>
-                                                    <h4 className="text-center">{currentLot.address_full}</h4>
+                                                    <h4 className="text-center">Records indicate an outstanding exactions balance of</h4>
+                                                    <div className="row text-center alert alert-danger">
+                                                        <h2><strong>{(currentLot.lot_exactions.current_exactions).toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',')}</strong></h2>
```
Comment (KellyInnovation) [inline lfucg-exactions/client/js/components/LotSummary.js]: Does current_exactions not have the $ and commas?

**Comment by JMStewart00** [comment]
It may not have when I did this the first time.

**Comment by JMStewart00** [comment]
@KellyInnovation - made some changes. Let me know what you think there.

**Pull request merged**